chart/templates/role.yaml

```diff
@@ -94,3 +183,22 @@ rules:
       - update
       - patch
 {{- end }}
+{{- end }}
+
+---
+# need clusterrole anyways
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "druid-operator.fullname" $ }}-sc
+  labels:
+    {{- include "druid-operator.labels" $ | nindent 4 }}
+rules:
+  - apiGroups:
+      - storage.k8s.io
+    resources:
+      - storageclasses
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
@zhangluva on which cloud platform was this needed, what was the k8s version ?

Reply:
Tested on EKS, `v1.22.11-eks-18ef993`

Reply:
@zhangluva just the indentation. Rest LGTM Thanks